### PR TITLE
Update @kiwicom/relay to latest version

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by FilipMessa** [comment]
@JoseDuSV thanks I didn't realize that, because I didn't check CI pipeline, I just run CI's tests on my local machine and it passes.

**Comment by tbergq** [comment]
Fixed the crash, now there are some flow errors originating from `@kiwicom/relay` that needs to be fixed

**Comment by mrtnzlml** [comment]
> Fixed the crash, now there are some flow errors originating from `@kiwicom/relay` that needs to be fixed

@tbergq These errors are actually caused by Margarita setup. Here is Git patch to fix it:

```patch
diff --git a/.flowconfig b/.flowconfig
index 6ea077e..22e66bc 100644
--- a/.flowconfig
+++ b/.flowconfig
@@ -26,9 +26,6 @@
 <PROJECT_ROOT>/node_modules/reqwest/.+
 <PROJECT_ROOT>/node_modules/fbemitter/.+
 <PROJECT_ROOT>/node_modules/xdl/build/detach/.+
-.*/node_modules/react-relay/lib/.+
-.*/node_modules/relay-compiler/lib/.+
-<PROJECT_ROOT>/node_modules/relay-runtime/lib/.+
 <PROJECT_ROOT>/node_modules/react-native-web/.+
 
 ; Generated by storybook
```

However, even with this fix please wait for me. I see some mistakes in this PR but you have to wait for a new `@kiwicom/relay` version. Shouldn't block you in any way, this PR can wait... :)

**Comment by tbergq** [comment]
It should work now, but let's wait for @mrtnzlml's new version of `@kiwicom/relay`, since I am not sure what mistakes he is referring to.

File: package.json [hunk 1]
```diff
@@ -59,7 +58,7 @@
     "jest-fetch-mock": "^2.0.1",
     "react-native-testing-library": "^1.4.2",
     "react-test-renderer": "^16.7.0",
-    "relay-compiler": "^1.7.0"
+    "relay-compiler": "^2.0.0"
```
Comment (mrtnzlml) [inline package.json]: Pls remove this - it should all be distributed in `@kiwicom/relay` 0.8.0.

File: packages/relay/package.json [hunk 2]
```diff
@@ -5,8 +5,11 @@
...
-    "@kiwicom/relay": "^0.1.0",
-    "react-relay": "^1.7.0"
+    "@kiwicom/relay": "^0.6.0",
+    "relay-runtime": "^2.0.0",
+    "react-relay": "^2.0.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "babel-plugin-relay": "^2.0.0"
+  }
```
Comment (mrtnzlml) [inline packages/relay/package.json]: This should all be removed and you should keep only `"@kiwicom/relay": "^0.8.0",`. The babel plugin may be necessary if you do not use `@kiwicom/babel-preset-kiwicom` (which I highly recommend).

**Comment by mrtnzlml** [comment]
Also, I do not see the `.flowconfig` changes? 🤔

**Comment by tbergq** [comment]
> Also, I do not see the `.flowconfig` changes?

They have already been merged.

File: packages/relay/package.json (hunk 2 above)
Comment (tbergq) [reply packages/relay/package.json]: I will look into this in a separate PR, since it seems to crash all our tests.

**Comment by tbergq** [comment]
@JoseDuSV @FilipMessa @mrtnzlml 🚀

File: packages/relay/package.json (hunk 2 above)
Comment (tbergq) [reply packages/relay/package.json]: Ok, looks like I solved it, so I will add it to this PR.

**Comment by FilipMessa** [comment]
Thanks 👍

**Pull request merged**